bootstrap/internal/controller/agentbootstrapconfig_controller.go

```diff
@@ -269,8 +269,7 @@ func (r *AgentBootstrapConfigReconciler) Reconcile(ctx context.Context, req ctrl
 		)
 		return ctrl.Result{}, err
 	}
-
-	log.Info("debug", "ns", config.Namespace)
+	log.V(logutil.TraceLevel).Info("metal3machine has ISO url", "abc", config)
```

Comment:
Should we print the full config here or maybe just print the metal3machine's iso url?